tests/test_flat.py

```diff
@@ -141,6 +170,18 @@ def test_roundtrip(tmp_path: pathlib.Path):
     roundtrip_helper(tmp_path, patients, "parquet", 4)
 
 
+@pytest.mark.skipif(meds_etl_cpp is None, reason="meds_etl_cpp not installed")
+def test_roundtrip_native(tmp_path: pathlib.Path):
+    meds_dataset = tmp_path / "meds"
+    create_dataset(meds_dataset)
+    patients = pq.read_table(meds_dataset / "data" / "patients.parquet").to_pylist()
+    patients.sort(key=lambda a: a["patient_id"])
+
+    roundtrip_helper(tmp_path, patients, "parquet", 1, backend="cpp")
```

Comment:
```suggestion
    roundtrip_helper(tmp_path, patients, format="parquet", num_proc=1, backend="cpp")
```

Reply:
Just because it's hard to keep 5 different arguments in your head as you're reading this